src/lambda/wizards/samInitWizard.ts

```diff
@@ -115,7 +115,7 @@ export class DefaultCreateNewSamAppWizardContext extends WizardContext implement
             items: templates.toArray().map(template => ({
                 label: template,
                 alwaysShow: template === currTemplate,
-                description:
+                detail:
```

Comment:
What does this change change?

Reply:
Just the way how description is displayed on the wizard. Now it is displayed under the template label, before it was right next to the label.  
![image](https://user-images.githubusercontent.com/54083696/73579037-1b969e80-4436-11ea-8cff-512c6774c218.png)